### PR TITLE
update ReadMe doc

### PR DESCRIPTION
**Pull request by linglp**
This PR is for updating ReadMe documentation #602. During my review of our current documentation, I found out that some of our command line options are not consistent. For example, for validating manifests, we use "-js, --json_schema" for json schema path while for getting manifests, we use " -j, --json_schema" for the json schema path. In addition, I tried to separate required options and optional options for CLI. Please help me double check and make sure that I listed them correctly!

### CODE REVIEW

**Comment by mialy-defelice** [comment]
Hey @linglp this looks great so far!

A few things I would change:

- [ ] Add bolded headers to separate instructions for `.synapseConfig` and `config.yml` at Step 5.
- [ ]  Move CLI instructions after the dev install instructions
- [ ] In the `config.yml` section, make it clear the user does not need to change that section if they will not be interacting with Synapse (people get really confused about this section).
- [ ] The bolding in section `Reporting bugs or feature requests` feels unnecessary.

Overall it looks fantastic!

**Comment by linglp** [comment]
> Hey @linglp this looks great so far!
> 
> A few things I would change:
> 
> * [x]  Add bolded headers to separate instructions for `.synapseConfig` and `config.yml` at Step 5.
> * [x]   Move CLI instructions after the dev install instructions
> * [x]  In the `config.yml` section, make it clear the user does not need to change that section if they will not be interacting with Synapse (people get really confused about this section).
> * [x]  The bolding in section `Reporting bugs or feature requests` feels unnecessary.
> 
> Overall it looks fantastic!

Thanks @mialy-defelice ! I have made the changes.

File: README.md [hunk 1]
```diff
@@ -1,60 +1,266 @@
...
+For obtaining `schematic_service_account_creds.json`
 ```
-schematic manifest --config ~/path/to/config.yml get    # generate manifest based on data type
+schematic init --config ~/path/to/config.yml  -a token
 ```
+## Other command line functions
+Please visit more documentation [here](https://sage-schematic.readthedocs.io/en/develop/cli_reference.html)
+
+
+# Release process
```
Comment (milen-sage) [inline README.md:206]: Move this section on release docs into RELEASE.md

File: README.md [hunk 2]
```diff
@@ -1,60 +1,266 @@
...
+# Other Contribution Guidelines
+### Reporting bugs or feature requests
+You can use the [`Issues`](https://github.com/Sage-Bionetworks/schematic/issues) tab to **create bug and feature requests**. Providing enough details to the developers to verify and troubleshoot your issue is paramount:
+- **Provide a clear and descriptive title as well as a concise summary** of the issue to identify the problem.
+- **Describe the exact steps which reproduce the problem** in as many details as possible.
+- **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+- **Explain which behavior you expected to see** instead and why.
+- **Provide screenshots of the expected or actual behaviour** where applicable.
+
+# Command Line Usage
```
Comment (milen-sage) [inline README.md:176]: You can remove this section.

**Review by milen-sage — CHANGES_REQUESTED** [review]
Left some comments requesting changes - but this is getting there!

**Review by milen-sage — APPROVED** [review]

**Comment by milen-sage** [comment]
To me this PR looks good and provides benefit to users - thanks @linglp !

@mialy-defelice we can merge and iterate on changes further later, unless you think more changes are needed here?

**Pull request merged**